### DISCUSSION TITLE
Which pins for ESP32-S3-DevKitC-1?

### DISCUSSION THREAD

**Discussion by kami83** [emsesp/EMS-ESP32#1842] [General]
Hi,

i am trying to connect the EMS interface board V3.1 to a ESP32-S3-DevKitC-1 with 16mb. I flashed it as lolin_s3. It is working. But which pins i should use for TXD and RXD?

BR kami

**Comment by proddy** [comment]
You can use any GPIO. See https://emsesp.github.io/docs/Configuring/#board-profile

Try the default ones we use for the S3 which are 5 and 17. You can see these also in the WebUI when you select the S32S3 board profile (and go to Custom...)

**Reply by kami83** [reply]
Hi, thanks a lot. Works fine. BR kami
